python/pyspark/sql/connect/plan.py

```diff
@@ -319,9 +319,9 @@ def __init__(
 
     def _convert_measure(
         self, m: MeasureType, session: Optional["RemoteSparkSession"]
-    ) -> proto.Aggregate.Measure:
+    ) -> proto.Aggregate.AggregateFunction:
         exp, fun = m
-        measure = proto.Aggregate.Measure()
+        measure = proto.Aggregate.AggregateFunction()
```

Comment:
This is the only code change that is caused by the proto change. We are lacking of aggregate testing coverage thus there is no tests affected. I will follow up on the testing coverage but that is a different problem.

cc @zhengruifeng